site/src/components/Auth0Provider.js

```diff
@@ -59,6 +60,8 @@ export const Auth0Provider = ({
 
       if (localIsAuthenticated) {
         setUser(await auth0FromHook.getUser())
+        const idToken = await auth0FromHook.getIdTokenClaims()
+        localStorage.setItem("idToken", idToken.__raw)
```

Comment:
Given `AuthorizedApolloProvider` is wrapped in the `Auth0Provider` I think we can avoid putting this in localstorage.